jekyll/_cci2/server/v4.1/installation/phase-1-prerequisites.adoc

```diff
@@ -0,0 +1,889 @@
+        "Status": "Active",
+        "CreateDate": "2017-07-31T22:24:41.576Z",
+        "SecretAccessKey": <AWS_SECRET_ACCESS_KEY>,
+        "AccessKeyId": <AWS_ACCESS_KEY_ID>
+  }
+}
+----
+--
+
+endif::env-gcp[]
+
+ifndef::env-aws[]
+
+[#google-cloud-storage]
+=== Google Cloud Storage
+
+[#create-a-gcp-bucket]
+==== a. Create a GCP bucket
+If your server installation runs in a GKE cluster, ensure that your current IAM user is cluster admin for this cluster, as RBAC (role-based access control) objects need to be created. More information can be found in the https://cloud.google.com/kubernetes-engine/docs/how-to/role-based-access-control[GKE documentation].
+
+[source,shell]
+----
+gsutil mb gs://circleci-server-bucket
+----
+
+[#set-up-authentication-gcp]
+==== b. Set up authentication
+The recommended method for workload/pod authentication is to use https://cloud.google.com/kubernetes-engine/docs/how-to/workload-identity[Workload Identity]. However, you may also use static credentials (json key file).
+
+. Create a Service Account.
++
+[source,shell]
+----
+gcloud iam service-accounts create circleci-storage --description="Service account for CircleCI object storage" --display-name="circleci-storage"
+----
+
+. Bind the `objectAdmin` role to the service account.
++
+[source,shell]
+----
+gcloud projects add-iam-policy-binding <PROJECT_ID> \
+    --member="serviceAccount:circleci-storage@<PROJECT_ID>.iam.gserviceaccount.com" \
+    --role="roles/storage.objectAdmin" \
+    --condition='expression=resource.name.startsWith("projects/_/buckets/circleci-server-bucket"),title=restrict_bucket'
+----
+
+. Either enable Workload Identity or use static credentials.
++
+[.tab.gcpauth.Workload_Identity]
+--
+**Option 1:** Workload Identity
+
+When using Workload Identity you need to configure your account such that the workloads/pods can access the storage bucket from the cluster using the Kubernetes service account `"<K8S_NAMESPACE>/object-storage"`.
+
+[source,shell]
+----
+gcloud iam service-accounts add-iam-policy-binding circleci-storage@<PROJECT_ID>.iam.gserviceaccount.com \
+    --role roles/iam.workloadIdentityUser \
+    --member "serviceAccount:<PROJECT_ID>.svc.id.goog[<K8S_NAMESPACE>/object-storage]"
+----
+
+[source,shell]
+----
+gcloud projects add-iam-policy-binding <PROJECT_ID> \
+    --member serviceAccount:circleci-storage@<PROJECT_ID>.iam.gserviceaccount.com \
+    --role roles/iam.serviceAccountTokenCreator \
+    --condition=None
+----
+--
+
+[tab.gcpauth.Static_credentials]
+--
+**Option 2:** Static credentials
+
+If you are not using Workload Identity, create a json file containing static credentials.
+
+[source,shell]
+----
+gcloud iam service-accounts keys create <KEY_FILE> \
+    --iam-account circleci-storage@<PROJECT_ID>.iam.gserviceaccount.com
+----
+--
+endif::env-aws[]
+
+ifndef::pdf[]
+[#next-steps]
+== Next steps
+* xref:server/v4.1/installation/phase-2-core-services#[Phase 2: Core Services Installation].
```

Comment:
@rsylvian here's an example of a cross reference. This works fine to get to: `circleci.com/docs/server/v4.1/installation/phase-2-core-services/` but fails the html proofer